NuRadioReco/framework/parameters.py

```diff
@@ -62,7 +62,7 @@ class channelParameters(Enum):
     block_offsets = 18 #: 'block' or pedestal offsets. See `NuRadioReco.modules.RNO_G.channelBlockOffsetFitter`
     Vrms_NuRadioMC_simulation = 19  #: the noise rms used in the MC simulation
     bandwidth_NuRadioMC_simulation = 20  #: the integrated channel response (=bandwidth for signal chains without amplification) used in the MC simulation
-
+    Vrms_trigger_NuRadioMC_simulation = 21  #: the noise rms of the trigger channels (otional) used in the MC simulation
```

Comment:
typo (optional).

Also, I don't know if this is the best place for it, but for non-RNO-G experts, it might be nice to know why this is optional, and when it is/isn't used?